fidesctl/src/fidesctl/core/config/__init__.py

```diff
@@ -47,4 +50,5 @@ def get_config(config_path: str = "") -> FidesctlConfig:
             except IOError:
                 echo_red(f"Error reading config file from {file_location}")
     fidesctl_config = FidesctlConfig()
+    echo_red("No config file found, using default values!")
```

Comment:
I figured this would be a handy message to throw users so they'll know right off the bat if their config file isn't getting ingested, but does this seem like too much?

Reply:
I think it's a great idea to include this! I'm just picky about the coloring and copy; since this doesn't indicate an error, it shouldn't be red. Once we add more robust log output to the CLI, I'd suggest we log all of the config values being used at the DEBUG level on startup. This is something I forgot to include in the API logging changes, but we should definitely add it there as well.

```suggestion
    print("No fidesctl.toml file found. Using default configuration values.")
```